forum/qa-plugin/report-reason/script.js

```diff
@@ -45,51 +34,40 @@ $(document).ready(function()
                  cache: false,
                  success: function(data)
                  {
-                    if(typeof data.error !== "undefined")
+                    if(data.error)
                     {
-                        console.log(data.error);
+            
+                        errorText = 'Blad serwera. Prosze sprobowac za jakis czas';
+                        console.log(data);
                         alert('Blad serwera. Prosze sprobowac za jakis czas');
                     }
-                    else if(typeof data.success !== "undefined")
+                    else if(data.success)
                     {
                         // if success, reload page
                         location.reload();
                     }
                     else
                     {
-                        console.log(data);
+                        errorText = 'Blad serwera. Prosze sprobowac za jakis czas';
                     }
                  },
                  error: function(data)
                  {
+                    errorText = 'Blad serwera. Prosze sprobowac za jakis czas';
                     alert('Wystapil blad! Przepraszamy za niedogodnosci');
                     console.log(data);
                  }
-            });        });
-        
-    }); // END click
+            });
     
-    // submit by enter key, cancel by escape key
-    $('.qa-flag-reasons-wrap').on('keyup', function(e)
-    {
-        console.log(e.keyCode);
-        if(e.keyCode == 13)
-        {
-            $(this).find('.qa-go-flag-send-button').click();
-        }
-        else if(e.keyCode == 27)
-        {
-            $(this).find('.closer').click();
-        }
-    });
-    
-    // mouse click on flagbox closes div
-    $('#flagbox-popup').click(function(e)
-    {
-        if(e.target == this)
-        { 
-            $(this).find('.closer').click();
-        }
+          if(errorText != '')
+          {
+              const errorHtml = document.createElement('p');
+              errorHtml.html = errorText;
```

Comment:
Elementy DOM nie posiadają property `html`, to pewnie pozostałość po zastąpieniu jQuery. Natywnie dostępne jest `innerHTML` i tego użyj.